### DISCUSSION TITLE
how to use globalTeardown idea?

### DISCUSSION THREAD

**Discussion by davidmanoukian** [vitest-dev/vitest#5686] [General]
Is there way to setup config in vitest.config.ts file to run the some functions after the all tests run?

**Comment by hi-ogawa** [comment]
You can use `globalSetup` https://vitest.dev/config/#globalsetup. Either you `export function teardown` https://github.com/vitest-dev/vitest/blob/main/test/global-setup/globalSetup/named-exports.js or `export default` https://github.com/vitest-dev/vitest/blob/main/test/global-setup/globalSetup/default-export.js
